### ISSUE TITLE
improve logger feature

### ISSUE THREAD

**Issue by quoeamaster** [quoeamaster/queutil#2]
added method to log message based on logging level
* if the logger's level is more significant comparing with the logging method (e.g. logLevel is INFO, and the logging method is Debug); then the message is NOT logged. Vice versa.
* so now the logger's level could be set / get

**Comment by quoeamaster** [comment]
code sample
```golang
// set Logger level to INFO
loggerForLevelTest.LogLevel = queutil.LogLevelInfo

// log a debug msg which would simply do nothing (ignored)
_, err = loggerForLevelTest.Debug([]byte(msg + "\n"))

// log an INFO msg which works
_, err = loggerForLevelTest.Info([]byte(msg + "\n"))

// log an WARN msg which works as well
_, err = loggerForLevelTest.Warn([]byte(msg + "\n"))

```

**Comment by quoeamaster** [comment]
feature released at **v0.1.2**
